examples/next-js/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "rm -rf .next && next dev",
```

Comment:
You want to keep that? If so I would suggest using `predev: 'rimraf .next'`

Reply:
Good catch, I'll remove that.